guides/spin/comparison/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+title:  "Spin and Roer + Managed Pipeline Templates"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Foreward
+
+This document is meant to describe the current state of managing
+pipeline and application config in Spinnaker 'as is', and describe
+a bit about future threads of work in this area.
+
+## `roer` and Managed Pipeline Templates
+
+### What it is
```

Comment:
A two-column table seems like the ideal format for this. I know Markdown isn't great with tables, but perhaps in this case it's worth "shelling out" to some HTML?

/cc @dorbin